.gitignore

```diff
@@ -28,4 +28,7 @@ build/Release
 node_modules
 
 # package-lock
-package-lock.json
\ No newline at end of file
+package-lock.json
+
+# generated files
+queue.mjs
```

Comment:
If you're adding to `.gitignore` I wouldn't expect the file to be checked in.

Also, what about the `.map` file?

Reply:
oh whoops. I think I created it and then I wanted to find out how @mcollina deploys this / how he expects people to use it - i.e. direct from github on occasion, etc, to determine if it is built and checked in, or built at deploy time.

The ci config only runs tests, there doesn't seem to be any automated deployment.